apiary.apib

```diff
@@ -1505,6 +1509,20 @@ Unlink a contact from a company.
 
 + Response 204
 
+### contacts.updateCompanyLink [POST /contacts.updateCompanyLink]
```

Comment:
Not sure about the naming but can't really think about any other?

Thinking out loud, but maybe we could allow `.linkToCompany` to do an upsert to facilitate this instead?

What are our thoughts about this?

Reply:
You can only link once imo. `SaveLinkToCompany` would do the trick, but is basically what we are doing here. (nothing against your proposal, just my 2 cents)

Reply:
It was considered during grooming to allow `.linkToCompany ` to become an upsert. 
Resonating to dismiss it was that this would change the current endpoint behaviour. 

Up until now, any duplicated call to `.linkToCompany` would fail due to an already existing link.  
After this, it would persist the new link properties.